lib/matplotlib/axes/_axes.py

```diff
@@ -4647,7 +4647,7 @@ def hexbin(self, x, y, C=None, gridsize=100, bins=None,
             colormapped rectangles along the bottom of the x-axis and
             left of the y-axis.
 
-        extent : float, default: *None*
+        extent : 4-tuple of float, default: *None*
             The limits of the bins. The default assigns the limits
             based on *gridsize*, *x*, *y*, *xscale* and *yscale*.
```

Comment:
if you are fixing, this maybe be explicit since we have at least 4 different ways to assign extents in the library:
```suggestion
            The limits of the bins (xmin, ymax, ymin, ymax) . The default assigns the limits
            based on *gridsize*, *x*, *y*, *xscale* and *yscale*.
```

Reply:
thanks, that makes sense. I probably violated some 79 char line width rule now by committing, right?

Reply:
That's actually already there in the third paragraph. If we move it here, then that line can be deleted.

Reply:
thanks, removed in 569336b